src/pages/workspaces/workspace/analysis/modals/CloudEnvironmentModal.js

```diff
@@ -207,9 +207,9 @@ export const CloudEnvironmentModal = ({
           shape: 'pause',
           toolLabel,
           onClick: () => stopApp(toolLabel),
-          disabled: busy || !canCompute,
+          disabled: cloudProvider === cloudProviderTypes.AZURE || busy || !canCompute, //TODO: IA-3993 enable pausing
           messageChildren: [span('Pause')],
-          tooltip: canCompute ? 'Pause Environment' : noCompute
+          tooltip: (cloudProvider === cloudProviderTypes.AZURE) ? 'Feature coming soon' : (canCompute ? 'Pause Environment' : noCompute)
```

Comment:
Nit: Realizing that this is two chained ternaries as well, this might be another candidate for a `Utils.cond`